blog/content/post/2023-01-23-3mdeb-insights-winter-2023.md

```diff
@@ -0,0 +1,122 @@
+ago and expand on it by describing examples on Rockchip and Mediatek-based
+boards. This talk will help developers integrate Secure Boot into their
+platforms for increased security in embedded devices.
+
+Maciej Pijanowski will discuss the TwTPM project, which aims to increase the
+trustworthiness of TPM modules by providing open-source firmware compliant
+with TCG PC Client Specification. TPMs enable security features such as measured
+boot and verified boot, but the market is dominated by proprietary firmware.
+The presentation will detail the project's goals, design, challenges, and
+status, also to gather a community and exchange ideas.
+
+Michał Żygowski will cover recent news, updates from previous years,
+and the current situation of coreboot. The talk will also discuss new
+deprecations and the shift of platforms out of the main tree.
+Michał will be discussing the history of AMD's cooperation in coreboot projects
+and the current state of support for modern mobile Ryzen chips on Chromebook
+devices. They will also present a plan for long-term, stable and sustainable
+support for older platforms. The talk will cover the efforts of 3mdeb to
+support the AMD platforms and the situation of maintaining PC Engines platforms.
+
+![dasharo-logo](/img/Dasharo-logo.svg)
+
+### Dasharo User Group (DUG) #1
+
+The Dasharo User Group (DUG) is an important forum for users of Dasharo to
+come together, share their knowledge, and stay informed about the latest
+developments in the Dasharo ecosystem. The DUG is a platform for users to
+connect and learn about new features and updates that are coming to Dasharo.
+The first DUG event will take place in early March and will include a variety
+of discussions on different topics related to Dasharo. The agenda for the event
+will be shared in the next month. The event will be a great opportunity for
+Dasharo users to meet other users, learn new things, and share their
+knowledge and experience with others.
+
+### Dasharo Developers vPub 0x6
+
+The Dasharo Developers vPub will be a follow-up event to DUG#1 and will
+provide a space for developers to engage in more informal conversations
+and discussions that may not have been covered during DUG#1. The vPub is
+designed to be a less structured, more relaxed environment where developers
+can discuss topics that are of interest to them. This can include off-topic
+discussions, technical challenges they are facing, and ideas for new features
+or improvements.
+
+The vPub is open to all developers who are interested in Dasharo and want to
+learn more about the platform. If you're curious about what was discussed in
+previous vPubs, you can find a list of past topics on the Dasharo vPub
+[archive page](https://vpub.dasharo.com/archive/). This will give you an idea
+of the types of conversations that take place during the vPub and help you
+decide if it's something you would like to participate in.
```

Comment:
I think whole description should tied even more to vPub history. Mention history of vPub little bit, please note first vPub was after first virtual FOSDEM and this is how it started.

Reply:
I added story to this event. 083a71e